src/Globals.cpp

```diff
@@ -37,6 +37,11 @@ net::websocket::SingleClientWSServer websocketServer("DefaultServer",
 													 Constants::WS_SERVER_PORT);
 std::atomic<bool> E_STOP = false;
 std::atomic<bool> AUTONOMOUS = false;
+
+// The boolean here is whether or not wheel rotation check should be ignored when doing
+// swerve-based driving
+std::tuple<DriveMode, bool> driveMode = {DriveMode::Normal, false};
```

Comment:
Same thing here as Abhay said. `std::pair` instead of `std::tuple` if you make that change in `Globals.h`.